### DISCUSSION TITLE
[Enhancement] [cascader] 级联选择器可自定义Tag

### DISCUSSION THREAD

**Discussion by jiaicon** [element-plus/element-plus#18234] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `cascader`

## Description

目前级联选择可自定义的部分太少，希望可以完善一下：
1. 可自定义后缀图标
2. 可自定义Tag

可参考Select组件的suffix-icon和slot

### Additional comments

_(empty)_

**Comment by warmthsea** [comment]
欢迎 PR
